A72-open-telemetry-tracing.md

```diff
@@ -36,98 +38,125 @@ and OpenCensus's binary format for (de)serialization. The header name is unique
 from other census library propagators to differentiate with the application’s 
 tracing instrumentation.
 
-The following tracing information during an RPC lifecycle should be captured:
+### Related Proposals and Documents:
+* [gRFC L29: C++ API Changes for OpenCensus Integration][L29]
+* [gRFC A45: Exposing OpenCensus Metrics and Tracing for gRPC retry][A45]
+* [Microservices observability overview][grpc-observability-public-doc]
+* [gRFC A66: OpenTelemetry Metrics][A66]
+
+## Proposal
+### gRPC OpenTelemetry Tracing API
+We will add tracing functions in grpc-open-telemetry plugin, along with OpenTelemetry
+metrics [gRFC A66][A66]. Languages will keep using gRPC infrastructures, e.g. interceptor and
+stream tracer to implement the feature, the same as Census.
+
+The OpenTelemetry API will coexist with the OpenCensus API. Only one
```

Comment:
Let's move this paragraph down to the "Migrate from OpenCensus to OpenTelemetry" section.